webapp/src/components/Header.test.tsx

```diff
@@ -21,7 +21,7 @@ const mockKeycloak = {
 
 describe('<Header />', () => {
   beforeEach(() => {
-    useKeycloak.mockReturnValue([mockKeycloak])
+    ;(useKeycloak as any).mockReturnValue([mockKeycloak])
```

Comment:
What does this leading ";" do?

Reply:
I have no idea, but prettier really wants it to be there haha

Reply:
Not affecting functionality, we can leave in for now